chapter_3/ex3_floating.rb

```diff
@@ -0,0 +1,31 @@
+# Tell ruby to count chickens
+puts "I will now count my chickens:"
+
+# Add 25 plus 30 hens then divide by 6
+puts "Hens #{25.0 + 30.0 / 6.0}"
```

Comment:
No need to fix, but FYI, if any of the numbers are floating point, the whole expression uses floating point math.  So `25 + 30 / 6.0` would work here.